tonic/src/transport/server/mod.rs

```diff
@@ -98,11 +99,13 @@ where
     B::Error: Into<crate::Error> + Send,
 {
     type Response = Response<BoxBody>;
+    type Error = crate::Error;
+
+    #[allow(clippy::type_complexity)]
```

Comment:
This is like so simple, clippy why 😭

Reply:
I tried to split them up into simpler type aliases but it makes them harder to read imo. We could add a `clippy.toml` increasing the threshold for this lint a bit and get rid of the allow annotations. I think there are 3 or so.